### DISCUSSION TITLE
Angluar SPA how are the angular files served

### DISCUSSION THREAD

**Discussion by Naliath** [dotnet/aspnetcore#48593] [Q&A]
I can turn off static file serving and everything keeps working locally. I tried blocking static file serving with an interceptor on the static file server but it does not seem to affect anything. 

What I want to do is block all requests  that are not authorized and channel this really simple site to Azure AD and after that I do not care about the identity.

No matter what I do I cannot seem to affect the serving of files. I found that for some inexplicable reason someone thought it was a great idea to have development processed differently than production. Unfortunately I have not been able to figure out  how I can intercept the SPA proxy configuration.

**Comment by gfoidl** [comment]
I guess you are aware of [Overview of Single Page Applications (SPA) in ASP.NET Core](https://learn.microsoft.com/en-us/aspnet/core/client-side/spa/intro?view=aspnetcore-7.0) (which explains the high level architecture here).

> want to do is block all requests that are not authorized

Did you see [Static file authorization](https://learn.microsoft.com/en-us/aspnet/core/fundamentals/static-files?view=aspnetcore-7.0#static-file-authorization)?

Note: the default SPA proxy is intended for some basic usage -- other use cases may need different setups, and on that behalf you can tweak the pipeline to your needs.

Also can you provide a (minimal) repo that showcases what you have tried?
Then it becomes easier to give concrete hints. So I'm a bit of sorry to just provide some general answer.
